src/app/components/external-link-access/external-link-access.component.ts

```diff
@@ -0,0 +1,26 @@
+import { Component, OnInit } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+import {Location} from '@angular/common';
+
+
+
+@Component({
+  selector: 'app-external-link-access',
+  templateUrl: './external-link-access.component.html',
+  styleUrls: ['./external-link-access.component.css']
+})
+export class ExternalLinkAccessComponent implements OnInit {
+
+  constructor(private activatedRoute: ActivatedRoute, private _location: Location) { }
+
+  public link: any;
```

Comment:
Can we provide the appropriate type here if it is not going to be type casted during the life cycle

Reply:
updated type as String